scripts/vm_mol_db_partitioning.py

```diff
@@ -0,0 +1,116 @@
+from time import time
+import pandas as pd
+import numpy as np
+from pathlib import Path
+from shutil import rmtree
+from concurrent.futures.thread import ThreadPoolExecutor
+
+CENTROIDS_BUCKET = 'omeruseast'
+MOL_DB_PREFIX = 'metabolomics/db/centroids_chunks'
+CENTROIDS_SEGMENTS_PREFIX = 'metabolomics/vm_db_segments'
+CENTR_SEGM_PATH = '/data/metabolomics/db/segms'
+DS_SEGMENTS = np.array([[79.99708557, 145.02574158], [145.02574158, 180.09922791], [180.09922791, 205.08337402],
```

Comment:
Just an FYI (I realize this is temporary code, this doesn't need to be fixed): `DS_SEGMENTS` has slight changes every time the segments are generated, because it's calculated based on a random sample of the dataset. If these values were actually used for searching the DS segments, this would be an issue.

Fortunately this file only uses the minimum, maximum and count of this array. Those numbers still change slightly between runs, but it has very little impact on the final data because there's usually no useful annotations at the min & max m/z.